source/team/vision.html.md.erb

```diff
@@ -1,9 +1,9 @@
 ---
 title: Our vision
-last_reviewed_on: 2021-01-13
+last_reviewed_on: 2021-01-27
 review_in: 3 months
 ---
 
 # <%= current_page.data.title %>
 
-A single platform to allow us to operate non-cloud-native MoJ services in a unified, scalable way, to allow us to more quickly migrate legacy services to the public cloud.
+A single platform which allows MoJ to modernise services, and run them in a unified, scalable way
```

Comment:
Great first PR @seanprivett 👍🏼

Could we expand out MoJ to Ministry of Justice and add a full stop?